### ISSUE TITLE
question about data

### ISSUE THREAD

**Issue by Shawn0099** [jackyko1991/vnet-tensorflow#8]
Hi,
I am using this code for brain tumor segmention(BRATS dataset),and  the result is not satisfying.In another issue you mentioned the data you uesd,i'm wondering if you can share the data so i can have a try.
Thanks

**Comment by jefflgaol** [comment]
http://academictorrents.com/details/27772adef6f563a1ecc0ae19a528b956e6c803ce
It is mentioned in README.md

**Comment by jackyko1991** [comment]
You may need to adjust the `config.json` for image patch size (for head data, i recommend to be around 256*256*32 with a 11GB GPU)

Also preprocessing pipeline need to be adjusted with different normalization methods, for LITS dataset you may try manual normalization with HU value [0,300], this is call soft tissue window and better visualizing specific tissues for both human and the machine

Several factors like decay rate, initial learning rate would also affect final result
